src/System.Management.Automation/resources/ParserStrings.resx

```diff
@@ -493,6 +493,9 @@ The correct form is: foreach ($a in $b) {...}</value>
   <data name="DuplicateScriptCommandClause" xml:space="preserve">
     <value>Script command clause '{0}' has already been defined.</value>
   </data>
+  <data name="MissingEndCurlyBraceOrNamedBlocks" xml:space="preserve">
+    <value>Missing closing '}' in script block. If you meant to add more statements to the script block, put the statements in one or more named blocks.</value>
```

Comment:
I don't think `named blocks` is a common term outside of the code, perhaps change the last part to: put the statements in begin, process, or end blocks.

Reply:
Fixed.